LayerClient/LayerClient.py

```diff
@@ -143,8 +152,11 @@ def _raw_request(self, method, url, data=None, extra_headers=None):
         )
 
         if result.ok:
-            return result.json()
-
+            try:
+                return result.json()
+            except JSONDecodeError:
```

Comment:
Doesn't `result.json()` throw a ValueError here? I think you should be able to just catch that, which saves us adding another dependency